packages/block-directory/src/components/downloadable-block-header/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { BlockIcon } from '@wordpress/block-editor';
+import BlockRatings from '../block-ratings';
+
+function DownloadableBlockHeader( { icon, title, rating, ratingCount, onClick } ) {
+	return (
+		<div className="block-directory-downloadable-block-header__row">
+			{
+				icon.match( /\.(jpeg|jpg|gif|png)$/ ) !== null ?
+					<img src={ icon } alt="block icon" /> :
+					<span >
+						<BlockIcon icon={ icon } showColors />
+					</span>
+			}
+
+			<div className="block-directory-downloadable-block-header__column">
+				<span role="heading" className="block-directory-downloadable-block-header__title" >
```

Comment:
The end-to-end AXE tests require an `aria-level` be provided when using `role="heading"`

https://dequeuniversity.com/rules/axe/3.5/aria-required-attr?application=axe-puppeteer

Normally it's not caught in end-to-end tests, but due to some unexpected circumstances, it's become an issue lately (#22712).

What is the intended level here?
Why can't it just be a semantic heading element, vs. `role=`?

See: #22713